src/Datadog.Trace/Util/UriHelpers.cs

```diff
@@ -21,18 +21,27 @@ public static string CleanUri(Uri uri, bool removeScheme, bool tryRemoveIds)
             return $"{uri.Scheme}{Uri.SchemeDelimiter}{uri.Authority}{path}";
         }
 
+        [Obsolete("This method is deprecated and will be removed. Use GetCleanUriPath() instead. " +
+                  "Kept for backwards compatability where there is a version mismatch between manual and automatic instrumentation")]
         public static string GetRelativeUrl(Uri uri, bool tryRemoveIds)
-        {
-            return GetRelativeUrl(uri.AbsolutePath, tryRemoveIds);
-        }
+            => GetRelativeUrl(uri.AbsolutePath, tryRemoveIds);
```

Comment:
Should we call `GetCleanUriPath()` to avoid the warning about calling an obsolete method?
```suggestion
            => tryRemoveIds ? GetCleanUriPath(uri.AbsolutePath) : uri.AbsolutePath;
```

Reply:
There is no warning because the method itself is `[Obsolete]` right?

Reply:
TIL. Never mind!